scripts/list_receipt_places.py

```
@@ -80,69 +109,45 @@ def list_all_metadatas(
     Args:
-        metadatas: List of ReceiptMetadata records
+        places: List of ReceiptPlace records
     """
-    if not metadatas:
-        print("No metadata records found.")
+    if not places:
+        print("No receipt-place records found.")
         return
 
     print(f"\n{'='*60}")
     print(f"SUMMARY")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)